src/openforms/submissions/logic/service_fetching.py

```diff
@@ -38,7 +50,13 @@ def perform_service_fetch(var: FormVariable, context: DataMapping) -> FetchResul
 
     client = fetch_config.service.build_client()
     request_args = fetch_config.request_arguments(context)
-    raw_value = client.request(**request_args)
+
+    cache_key = hash(submission_uuid + json.dumps(request_args, sort_keys=True))
+
+    raw_value = cache.get(cache_key, sentinel)
+    if raw_value is sentinel:
+        raw_value = client.request(**request_args)
+        cache.set(cache_key, raw_value)
```

Comment:
Discussed:
The cache headers will be picked up as part of a separate issue once we have reworked the ZDS clients (since in this case, only the JSON of the response is returned and not the headers).